Gemfile

```diff
@@ -86,4 +86,5 @@ group :test do
   gem 'orderly'
   gem "selenium-webdriver"
   gem "webdrivers"
-end
\ No newline at end of file
+end
+gem "sidekiq", "~> 7.1"
```

Comment:
Will we need to add any dependencies for Redis?

Reply:
Redis comes over as a dependency of the `sidekiq` gem: https://github.com/turingschool/present/blob/feat/sidekiq/Gemfile.lock#L291